src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -902,6 +898,40 @@ partial class BoundConditionalAccess : IConditionalAccessExpression
         protected override OperationKind ExpressionKind => OperationKind.ConditionalAccessExpression;
     }
 
+    partial class BoundEqualsValue : ISymbolInitializer
```

Comment:
It feels like this file contains to many type declarations. We should consider moving all IOperation- specific code into a separate file(s). One file per type under an IOperation folder would be preferred. This doesn't have to be done as part of this change, but sooner is better than later.